sklearn/decomposition/_dict_learning.py

```diff
@@ -579,11 +568,10 @@ def dict_learning(X, n_components, *, alpha, max_iter=100, tol=1e-8,
         dictionary = np.r_[dictionary,
                            np.zeros((n_components - r, dictionary.shape[1]))]
 
-    # Fortran-order dict, as we are going to access its row vectors
+    # Fortran-order dict better suited for the sparse coding which is the
+    # bottleneck of this algorithm.
```

Comment:
In the call below, it could be even more explicit to use `asfortranarray` instead of `array(..., order="F")`